bin/package.js

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env node
+process.removeAllListeners('warning');
+
+// ES Modules
+import {findUp} from "find-up"
```

Comment:
I thought about it but ended up not going that path because we'll end up including in `@shopify/cli-kit` dependencies and code that's only necessary for development automation. A good example of it is the octokit package.
I think if we continue adding scripts we should consider having an automation kit or a development toolkit.